src/models/scatter.js

```diff
@@ -256,6 +255,18 @@ nv.models.scatter = function() {
                         [width + 10,-10]
                     ]);
 
+                    // delete duplicates from vertices - essential assumption for d3.geom.voronoi
+                    var epsilon = 1e-4; // Uses 1e-4 to determine equivalence.
+                    vertices = vertices.sort(function(a,b){return ((a[0] - b[0]) || (a[1] - b[1]))});
```

Comment:
@liquidpele, I got inspiration from this P.R. https://github.com/novus/nvd3/pull/1934
It looks like the sort is used to make sure we dedupe properly below.